config/djangobb_requirements.txt

```diff
@@ -0,0 +1,10 @@
+django-haystack>=2.1.0,<2.4
+Pillow>=2.1.0
+postmarkup
+pygments
+django-pagination-py3==1.1.1
+django-allauth==0.19.1
+django-messages==0.5.1
+django-model-utils==2.2
```

Comment:
Not sure whether I like maintaining multiple requirements files, particularly since at least two of these are repeated (django-model-utils and Pillow)
@bethlakshmi any thoughts on this one?